bitnami/common/Chart.yaml

```diff
@@ -20,4 +20,4 @@ sources:
   - https://github.com/bitnami/charts
   - https://www.bitnami.com/
 type: library
-version: 1.15.2
\ No newline at end of file
+version: 1.16.0
```

Comment:
It's also needed to bump the appVersion in order to match this one, see some lines above